src/cookies.js

```diff
@@ -253,8 +253,9 @@ function checkOriginForSettingCookie(win, options, name) {
   );
   const current = parseUrlDeprecated(win.location.href).hostname.toLowerCase();
   const proxy = parseUrlDeprecated(urls.cdn).hostname.toLowerCase();
+  const metaTag = win.document.querySelector("meta[name='runtime-host']");
```

Comment:
would you mind adding some tests for this, mostly because i'd like to understand what the impact of this as far as cookie writes for origin domain vs cache

Reply:
Gotcha I'll go ahead and do that for ya!

Reply:
Makes since @mdmower I will push that change up right now. Thank you for the suggestion!

Reply:
@JakeSquier - It looks like you're still omitting the `head` existence check that I recommending mimicking from here: https://github.com/ampproject/amphtml/blob/cc0a63f686539f6b7a1064fd93c397e35c54a4ba/src/cookies.js#L119-L121 

Did you confirm that it was superfluous?

Reply:
I apologize I had mistaken what you meant previously. It was changed to only query the head but, I have now updated it to check for the heads existence as well as only query the head like you specified in the snippet of code you provided above. Now it is as so:
`const metaTag = win.document.head && win.document.head.querySelector("meta[name='runtime-host']");`